modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/auto_fields.js

```diff
@@ -454,10 +454,20 @@ AUI.add(
 
 					const contentBox = instance._contentBox;
 
+					let computedStyle;
+
```

Comment:
Is there a reason to have this variable outside of local scope? If no, it's a better practice to keep it in. Something like:
```js
const computedStyle = window.getComputedStyle(node);
```

Reply:
Not really. Thanks for the suggestion, @markocikos !